auth/auth-domain/src/main/java/shop/woowasap/auth/domain/exception/DuplicatedUserIdException.java

```diff
@@ -0,0 +1,10 @@
+package shop.woowasap.auth.domain.exception;
+
+public final class DuplicatedUserIdException extends AuthDomainBaseException {
+
+    private static final String MESSAGE = "유저의 아이디가 중복되었습니다. 입력 값 : ";
+
+    public DuplicatedUserIdException(String userId) {
+        super(MESSAGE + userId);
```

Comment:
예외 메시지를 아래와 같이 `"` 로 매핑하면, 공백을 쉽게 확인할 수 있어요.

아래 두 메시지는 모두 hello 앞에 공백이 있는데, 매핑되어 있지 않으면, 모른채로 지나치기 쉽습니다.
`"유저의 아이디가 중복되었습니다. 입력 값 :  hello"`
`"유저의 아이디가 중복되었습니다. 입력 값 : " hello""`

```suggestion
        super(MESSAGE + "\"" +userId + "\"");
```

Reply:
하나 배워갑니다. [fc8de95](https://github.com/woowasap/ASAP/pull/43/commits/fc8de9548760f00f3c50a873ff103e86a2b04a47)